src/pages/publication/PublicationPage.tsx

```diff
@@ -30,7 +67,72 @@ const PublicationPage: FC<PublicationPageProps> = ({ publicationId }) => {
   return (
-    <>{isLoadingPublication ? <CircularProgress color="inherit" size={20} /> : <p>Retrieved: {publication.id}</p>}</>
+    <>
+      {isLoadingPublication ? (
+        <CircularProgress color="inherit" size={20} />
+      ) : (
+        <ContentPage>
+          {publication?.id ? (
+            <>
+              <FormCardHeading>{publication.title?.no}</FormCardHeading>
+              {publication.authors && <PublicationPageAuthors authors={publication.authors} />}
+              <ContentWrapper>
+                <Sidebar>
+                  <StyledSidebarCard>
+                    <NormalText>NTNU institutt for osteloff</NormalText>
+                    <NormalText>SINTEF Teknologi og samfunn</NormalText>
+                  </StyledSidebarCard>
+                  {publication.files && (
+                    <StyledSidebarCard>
+                      <PublicationPageFiles files={publication.files} />
+                    </StyledSidebarCard>
+                  )}
+                  <StyledSidebarCard>
+                    <LabelContentRowForPublicationPage label={t('description.date_published')}>
+                      {publication.publicationDate.year}
+                      {publication.publicationDate.month && `-${publication.publicationDate.month}`}
+                      {publication.publicationDate.day && `-${publication.publicationDate.day}`}
+                    </LabelContentRowForPublicationPage>
+                    <PublicationPageIdentifiers publication={publication} />
+                  </StyledSidebarCard>
+                </Sidebar>
+                <MainContent>
+                  {publication.doiLink && (
+                    <LabelContentRowForPublicationPage label={t('references.doi')}>
+                      <Link href={publication.doiLink}>{publication.doiLink}</Link>
+                    </LabelContentRowForPublicationPage>
+                  )}
+                  {publication.abstract && (
+                    <LabelContentRowForPublicationPage label={t('description.abstract')}>
+                      {publication.abstract}
+                    </LabelContentRowForPublicationPage>
+                  )}
+                  {publication.description && (
+                    <LabelContentRowForPublicationPage label={t('description.description')}>
+                      {publication.description}
+                    </LabelContentRowForPublicationPage>
+                  )}
+                  {publication.tags && (
+                    <LabelContentRowForPublicationPage label={t('description.tags')}>
+                      {publication.tags}
+                    </LabelContentRowForPublicationPage>
+                  )}
+                  <PublicationPageJournal publication={publication} />
+                  {publication.projects.length > 0 && (
+                    <LabelContentRowForPublicationPage label={t('description.project_association')}>
+                      {publication.projects?.[0].titles?.[0].title}
```

Comment:
bruker bare `[0]` til datamodell er klar?

Reply:
jepp. For demo purposes